concent_api/core/tests/test_auth_force_payment.py

```diff
@@ -32,23 +32,23 @@ def test_provider_send_force_payment_and_concent_should_return_it_to_requestor_w
         """
         subtask_results_accepted_list = [
             self._get_deserialized_subtask_results_accepted(
-                timestamp       = "2018-02-05 10:00:15",
-                payment_ts      = "2018-02-05 11:55:00",
-                task_to_compute = self._get_deserialized_task_to_compute(
-                    timestamp                       = "2018-02-05 10:00:00",
-                    deadline                        = "2018-02-05 10:00:10",
-                    subtask_id='2',
-                    price                           = 15000,
+                timestamp="2018-02-05 10:00:15",
+                payment_ts="2018-02-05 11:55:00",
+                task_to_compute=self._get_deserialized_task_to_compute(
+                    timestamp="2018-02-05 10:00:00",
+                    deadline="2018-02-05 10:00:10",
+                    subtask_id=self._get_uuid(),
+                    price=15000,
```

Comment:
All style changes should be in separate commit :)

Reply:
as above- it should be, but for me it is readable and can stay..